index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="styles.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css" />
+    <title>Setup and Mibile first</title>
+</head>
+<body>
+    <header class="header">
+        <div class="header-container">
+            <a href="#" class="logo">My Logo</a>
+            <nav class="nav">
+                <i class="fa fa-bars"></i>
+            </nav>
+        </div>
+    </header>
+
+    <main class="main">
+        <section class="section-hero">
+            <div class="hero-container">
+                <h1>I’m Eugene Glad to see you!</h1>
+                <p>I’m a software developer! I can help you build a product , feature or website Look through some of my
+                    work and experience! If you like what you see and have a project you need coded, don’t hestiate to
+                    contact me.</p>
+                <div class="social-links">
+                    <h2><img src="image/LET’S CONNECT.svg" alt=""></h2>
```

Comment:
- [ ] It would be better is you use text instead of an image inside the `h2` tag. 😃